demos/smartlab_demo/python/smartlab_demo.py

```diff
@@ -16,54 +18,183 @@
+from segmentor import Segmentor, SegmentorMstcn
+from object_detection.detector import Detector
 
 
 def build_argparser():
     parser = ArgumentParser(add_help=False)
     args = parser.add_argument_group('Options')
     args.add_argument('-h', '--help', action='help', default=SUPPRESS,
-                    help='Show this help message and exit.')
+                      help='Show this help message and exit.')
     args.add_argument("-d", "--device", type=str, default='CPU', required=False,
-                        help="Optional. Specify the target to infer on CPU or GPU.")
+                      help="Optional. Specify the target to infer on CPU or GPU.")
     args.add_argument('-tv', '--topview', required=True,
-                    help='Required. Topview stream to be processed. The input must be a single image, '
-                    'a folder of images, video file or camera id.')
-    args.add_argument('-fv', '--frontview', required=True,
-                    help='Required. FrontView to be processed. The input must be a single image, '
-                    'a folder of images, video file or camera id.')
-    args.add_argument('-m_ta', '--m_topall', help='Required. Path to topview all class model.', required=True, type=str)
-    args.add_argument('-m_tm', '--m_topmove', help='Required. Path to topview moving class model.', required=True, type=str)
-    args.add_argument('-m_fa', '--m_frontall', help='Required. Path to frontview all class model.', required=True, type=str)
-    args.add_argument('-m_fm', '--m_frontmove', help='Required. Path to frontview moving class model.', required=True, type=str)
-    args.add_argument('-m_en', '--m_encoder', help='Required. Path to encoder model.', required=True, type=str)
-    args.add_argument('-m_de', '--m_decoder', help='Required. Path to decoder model.', required=True, type=str)
+                      help='Required. Topview stream to be processed. The input must be a single image, '
+                           'a folder of images, video file or camera id.')
+    args.add_argument('-sv', '--sideview', required=True,
+                      help='Required. SideView to be processed. The input must be a single image, '
+                           'a folder of images, video file or camera id.')
+    args.add_argument('-m_ta', '--m_topall', help='Required. Path to topview all class model.',
+                      required=True, type=str)
+    args.add_argument('-m_tm', '--m_topmove', help='Required. Path to topview moving class model.',
+                      required=True, type=str)
+    args.add_argument('-m_sa', '--m_sideall', help='Required. Path to sidetview all class model.',
+                      required=True, type=str)
+    args.add_argument('-m_sm', '--m_sidemove', help='Required. Path to sidetview moving class model.',
+                      required=True, type=str)
+    args.add_argument('--mode', default='multiview', help='Optional. action recognition mode: multiview or mstcn',
```

Comment:
Hi, it's because multiview has async_inference implemetation which has some bug and causes access the violation error. I remove the async action recognition implementation as a workaround because OMZ demo isn't aimed to have best performance. 

If user need high performance, they should use their own pipeline architecture or DL Streamer instead.